src/test/java/de/bonndan/nivio/input/dto/ItemDescriptionFactoryNivioTest.java

```diff
@@ -84,36 +82,38 @@ public void readServiceAndInfra() {
             }
         });
 
-        assertNotNull(service.getDataFlow());
-        assertEquals(3, service.getDataFlow().size());
-        service.getDataFlow().forEach(dataFlow -> {
-            if (dataFlow.getDescription().equals("kpis")) {
+        assertNotNull(service.getRelations(RelationType.PROVIDER));
+        assertEquals(3, service.getProvidedBy().size());
+
+        Set<RelationItem<String>> dataflows = service.getRelations(RelationType.DATAFLOW);
+        assertNotNull(dataflows);
+        assertEquals(3, dataflows.size());
+        dataflows.forEach(dataFlow -> {
+             if (dataFlow.getDescription().equals("kpis")) {
                 Assert.assertEquals("content-kpi-dashboard", dataFlow.getTarget());
             }
         });
 
-        ServiceDescription web = services.get(2);
-        assertEquals(ServiceItem.LAYER_INGRESS, web.getLayer());
+        ItemDescription web = services.get(2);
+        assertEquals(LandscapeItem.LAYER_INGRESS, web.getLayer());
         assertEquals("wordpress-web", web.getIdentifier());
         assertEquals("Webserver", web.getDescription());
         assertEquals("Apache", web.getSoftware());
         assertEquals("2.4", web.getVersion());
         assertEquals("Pentium 1 512MB RAM", web.getMachine());
         assertEquals("ops guys", web.getTeam());
         assertEquals("content", web.getNetworks().toArray()[0]);
```

Comment:
No, it might even be too coarse. The input in Nivio consists of DTOs called XXXDescription, mostly similar to the final objects in the graph. The tests makes sure that every aspect of a DTO is copied to its target object.